src/main/java/services/PriceCalculation.java

```diff
@@ -0,0 +1,85 @@
+package services;
+
+import entities.Flight;
+import entities.Plane;
+import entities.Ticket;
+
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+import java.util.*;
+
+
+public class PriceCalculation {
+    private final static Long BASIC_PRICE = 100L;
+    private final static Long BASIC_LUGGAGE_FEE = 50L;
+    private final static Long APPROACHING_DEPARTURE_DAILY_FEE = 35L;
+    private final static Long LUGGAGE_PRICE_PER_KILO = 5L;
+
+    public void setPriceInFlights(Set<Flight> flights,
+                                         Map<Flight, Ticket> ticketByFlight,
+                                         Map<Ticket, Long> luggageByTicket,
+                                         Map<Ticket, Boolean> priorityByTicket,
+                                         Map<Flight, Plane> planesByFlight,
+                                         /*Set<Ticket> setOfTickets,
+                                         Set<Plane> setOfPlanes,*/
+                                         Map<Plane, Double> loadByPlane) throws SQLException{
```

Comment:
IDK why but this maps looks like bad architecture, at list you can go to dao and get Plane, Tiket and load only having set<flight>